src/exchange/exchange.py

```diff
@@ -38,10 +39,10 @@ class Exchange:
     provider: Provider
     model: str
     system: str
-    moderator: Moderator = field(default=ContextTruncate())
+    moderator: Moderator = field(default=PassiveModerator())
```

Comment:
any reason for changing this to Passive? Truncate is probably a better user experience

Reply:
Ah didn't catch this in my proof-read - meant to change it back. Though thoughts on making it `ContextSummarize()`?

Reply:
Yea eventually. I think we still need feedback on it to see how well it's working for people

Reply:
Ok, just set it back to `ContextTruncate` and we can revisit later.